iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -640,6 +667,41 @@ - (void)updateBookmark:(MWMMarkID)bookmarkId
   }
 }
 
+- (void)moveBookmark:(MWMMarkID)bookmarkId
+           toGroupId:(MWMMarkGroupID)groupId {
+    auto const currentGroupId = self.bm.GetBookmark(bookmarkId)->GetGroupId();
+    auto editSession = self.bm.GetEditSession();
+    if (groupId != kml::kInvalidMarkGroupId) {
```

Comment:
Why is this check necessary? What is the case when groupId can be invalid here?
And if it's really necessary, then this check should be the first line in the function.

Reply:
There was similar check in `updateBookmark:setGroupId:title:color:description:` before `editSession.MoveBookmark`, so I made it the same.

Reply:
I would prefer to crash immediately to find the bug than to do nothing and silently ignore the error in the code.

Reply:
I'm not sure it will crash in such case, looks like it just will be moved to group with groupId=MAXINT. 
(I don't know much about framework internals and how it will work in such case)